docs/scout.txt

```diff
@@ -0,0 +1,254 @@
+         
+         return [
+             'driver' => env('SCOUT_DRIVER', 'mongodb'),
+             'mongodb' => [
+                 'connection' => env('SCOUT_MONGODB_CONNECTION', 'mongodb'),
+             ],
+             'prefix' => env('SCOUT_PREFIX', 'scout_'),
+         ];
+
+      The preceding code specifies the following configuration:
+      
+      - Uses the value of the ``SCOUT_DRIVER`` environment variable as
+        the default search driver, or ``mongodb`` if the environment
+        variable is not set
+        
+      - Specifies ``scout_`` as the prefix for the collection name of the
+        searchable collection
+      
+      In the ``config/scout.php`` file, you can also specify a custom
+      Atlas Search index definition. To learn more, see the :ref:`custom
+      index definition example <laravel-scout-custom-index>` in the
+      following step.
+
+      Set the following environment variable in your application's
+      ``.env`` file to select ``mongodb`` as the default search driver:
+      
+      .. code-block:: none
+         :caption: .env
+         
+         SCOUT_DRIVER=mongodb
+      
+      .. tip:: Queueing
+      
+         When using Scout, consider configuring a queue driver to reduce
+         response times for your application's web interface. To learn more,
+         see the `Queuing section
+         <https://laravel.com/docs/{+laravel-docs-version+}/scout#queueing>`__
+         of the Laravel Scout documentation and the :ref:`laravel-queues` guide.
+
+   .. step:: Create the Atlas Search index
+
+      After you configure Scout and set your default search driver, you can
+      create your searchable collection and search index by running the
+      following command from your application's root directory:
+        
+      .. code-block:: bash
+         
+         php artisan scout:index 'App\Models\Movie'
+        
+      Because you set MongoDB as the default search driver, the preceding
+      command creates the search collection with an Atlas Search index in your
+      MongoDB database. The collection is named ``scout_movies``, based on the prefix
+      set in the preceding step. The Atlas Search index is named ``scout``
+      and has the following configuration by default:
+
+      .. code-block:: json
+      
+         {
+           "mappings": {
+             "dynamic": true
+           }
+         }
+
+      .. _laravel-scout-custom-index:
+
+      To customize the index definition, add the ``index-definitions``
+      configuration to the ``mongodb`` entry in your
+      ``config/scout.php`` file. The following code demonstrates how to
+      specify a custom index definition to create on the
+      ``scout_movies`` collection:
+
+      .. code-block:: php
+      
+         'mongodb' => [
+             'connection' => env('SCOUT_MONGODB_CONNECTION', 'mongodb'),
+             'index-definitions' => [
+                 'scout_movies' => [
+                     'mappings' => [
```

Comment:
I find this page very useful about mappings: https://www.mongodb.com/docs/atlas/atlas-search/define-field-mappings/

Also the playground: https://www.mongodb.com/docs/atlas/atlas-search/playground/

Reply:
Will add links!